add LM3914V led driver

lorem-ipsum opened the pull request:
This adds the classical [LM3914](https://www.ti.com/lit/ds/symlink/lm3914.pdf) in the PLCC variant. Placed the REFOUT pin on the left as well as it separates the control side from LED connecting side.

![Screenshot_20190817_211617](https://user-images.githubusercontent.com/1010629/63216381-752f3900-c134-11e9-9e16-31780a830b8e.png)
---

All contributions to the kicad library must follow the [KiCad library convention](http://kicad-pcb.org/libraries/klc/)

Thanks for creating a pull request to contribute to the KiCad libraries! To speed up integration of your PR, please check the following items:

- [x] Provide a URL to a datasheet for the symbol(s) you are contributing
- [x] An example screenshot image is very helpful
- [x] Ensure that the associated footprints match the [official footprint library](https://github.com/kicad/kicad-footprints)
  - A new fitting footprint must be submitted if the library does not yet contain one.
- [x] If there are matching footprint PRs, provide link(s) as appropriate
- [x] Check the output of the Travis automated check scripts - fix any errors as required
- [x] Give a reason behind any intentional library convention rule violation.

myfreescalewebpage commented:
Hi @lorem-ipsum , thanks for contributing,

A few comments I have during my review:

- [ ] The description should ends with the complete package name : `10 LED Dot/Bar Display Driver, PLCC-20`
- [ ] The number of the pins correspond to the PDIP package, not the PLCC (see page 22 of the datasheet)

Cheers,
Joel

lorem-ipsum commented:
Hi @myfreescalewebpage ,

I updated the description and fixed the pin assignments. Thanks for spotting this, I completely ignored this -.-

![LM3914V](https://user-images.githubusercontent.com/1010629/81914851-1ec32e00-95d2-11ea-8d15-2d55158e1e8e.png)


Cheers,
Hendrik

myfreescalewebpage commented:
Thanks, no more comment, merging.
